messagebroker.js

```diff
@@ -24,8 +24,32 @@ MessageBroker.prototype.init = function () {
     setInterval(this.sendMessage.bind(this), 1500);
 };
 
+var messageBroker;
+
+function initialize(sock) {
+    messageBroker = new MessageBroker(sock);
```

Comment:
You might want to throw an exception here if `messageBroker` is already defined, so that you can't accidentally overwrite it.